sieve_of_eratosthenes/sieveOfEratosthenes.c

```diff
@@ -0,0 +1,34 @@
+#include <stdio.h>
+#include <string.h>
+#include <math.h>     // for using square root 
+#include <stdbool.h>  // to include bool datatype 
+
+void sieveOfEratosthenes(int n) {
+	bool primes[n+1];
+	int i, sqrt_n = sqrt(n)+1, j;
```

Comment:
No underscores for variable names.
It should follow `[a-z][a-z0-9][a-zA-Z0-9]*`